sycl/test/lit.cfg

```diff
@@ -98,3 +98,9 @@ path = config.environment['PATH']
 path = os.path.pathsep.join((config.llvm_tools_dir, path))
 config.environment['PATH'] = path
 
+# Set timeout for test = 10 mins
+try:
+    import psutil  # noqa: F401
+    lit_config.maxIndividualTestTime = 600
```

Comment:
How did you get 10 minutes?
Is it enough for slowest configuration: e.g. debug+sanitizers+1thread+<my_slowest_cpu>+...?

Reply:
Just took the same number as used for lldb testing. I think there is no problems with adjusting it if we encounter problems.
Note: the timeout is per LIT test, not for the whole suite.